float32 remaining_distance
# size of the array of gps coordinates
int32 gps_coordinates_size
